### DISCUSSION TITLE
p2g correlations and trajectory inference questions

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1454] [Questions / Documentation]
Hi @rcorces,

I am wondering what you think about these questions (would appreciate your expert opinion) -

1. Peak to gene correlations that are significantly negative (let's say R = -0.45) and pass the FDR threshold (FDR < 0.05). Do you think these could represent biologically plausible relationships (such as with TF repressor binding or silencer) or do you think they may be just technical noise?

2. In Trajectory analysis, do you think there is a way to compare whether two trajectories/lineages are statistically different? ArchR has a function to CorrelateTrajectories - is there a way to extrapolate the results from this to answer this question?

**Comment by rcorces** [comment]
1. Yes - I think these can be real.
2. I'm not aware of an easy way to do this. Sorry!
